### PR TITLE
Workflow Automated Test - LRQA-67659 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

**Comment by PauloVAS** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsCompletionVelocity.testcase [hunk 1]
```diff
@@ -75,12 +75,66 @@ definition {
...
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			panel = "Site Administration",
+			portlet = "Blogs");
+
+		Blogs.addWithWorkflow(
+			entryContent = "Test Content",
```
Comment (CalumR23) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsCompletionVelocity.testcase:103]: Could you please change this macro to 
`JSONBlog.addEntry( entryContent = "Blogs Entry Content", entryTitle = "Blogs Entry Title"); `

**Comment by PauloVAS** [comment]
@CalumR23, done!

**Comment by rodrigocunhaa** [comment]
ci:forward

**Pull request closed without merging**